### PR TITLE
Orbital into class

### PR DESCRIPTION
**Pull request by jinxcat**
addresses #13

### CODE REVIEW

**Comment by jinxcat** [comment]
@johngian please review

**Comment by johngian** [comment]
Please rebase `master` and modify branch appropriately in order to have a single commit. https://github.com/satnogs/satnogs-client/commit/3fa8c64d0b99460d68b37497c651ea1a13f68fd8 Is a merge commit from the same branch `orbital_into_class`.

**Comment by johngian** [comment]
Please rename https://github.com/satnogs/satnogs-client/commit/53d37eb67a78316f821c6b5c3e9478e4dca85795 to something more descriptive.

File: satnogsclient/observer/orbital.py [hunk 1]
```diff
@@ -1,56 +1,106 @@
 # -*- coding: utf-8 -*-
-import sys
-
 from datetime import datetime
-
-import ephem
 import pytz
+import ephem
```
Comment (johngian) [inline satnogsclient/observer/orbital.py:4]: Why are you changing the order. We generally want them alphabetically ordered (not required from PEP8 though).

Comment (jinxcat) [inline satnogsclient/observer/orbital.py:4]: I never heard before alphabetical ordering is important. In my mind, datetime and pytz go together on grounds of relevance

Comment (johngian) [inline satnogsclient/observer/orbital.py:4]: Many major python projects require alphabetical ordering. Here [1] is the most common style guide for module importing (not mozilla specific, many projects follow this). Also there is no need to change the order here (doesn't make any difference to the commit).

[1] https://mozweb.readthedocs.org/en/latest/reference/python-style.html#import-statements

**Comment by johngian** [comment]
Bump @jinxcat 
Could you please squash to continue with the revieew?

**Comment by johngian** [comment]
Closing this one after #40

**Pull request closed without merging**